src/lua/protobuf.lua

```diff
@@ -545,6 +581,39 @@ validate_scalar = function(field_def, value)
     end
 end
 
+local function validate_decoded_type(protocol, field_def, wire_type)
+    local protobuf_wire_types = {[0] = 'varint', [1] = 'I64',
+        [2] = 'len', [5] = 'I32'}
```

Comment:
Here we use the wireformat type IDs and define wireformat type names. Let's move it to the wireformat module as well.

TBH, while looking over the code it seems for me that there should be something like wireformat registry in the wirefomat module, similar to `scalars` here. So, we can do something like this:

```lua
wireformat.VARINT.id => 0
wireformat.VARINT.name => 'varint'
```

Reply:
This table is needed to correlate recieved integer numbers from decoding with protobuf types. I moved this table to wireformat module and made links to wireformat types from wireformat module.